crates/astria-sequencer/src/asset/mod.rs

```diff
@@ -0,0 +1,16 @@
+use once_cell::sync::OnceCell;
+use proto::native::sequencer::v1alpha1::asset::Denom;
+
+pub(crate) static NATIVE_ASSET: OnceCell<Denom> = OnceCell::new();
+
+pub(crate) fn initialize_native_asset(native_asset: &str) {
```

Comment:
Pair this with a `get_native_asset -> &Denom` function so that you don't have to `.expect()` at every call site.